core/bug_api.php

```diff
@@ -781,8 +789,26 @@ function update( $p_update_extended = false, $p_bypass_mail = false ) {
 			email_bug_updated( $c_bug_id );
 		}
 
+		# Execute all registered update callbacks
+		while( !empty( $this->update_callbacks ) ) {
+			$t_callback = array_shift( $this->update_callbacks );
+			call_user_func_array( $t_callback['func'], $t_callback['params'] );
+		}
+
+		event_signal( 'EVENT_UPDATE_BUG', array( $t_old_data, $this ) );
```

Comment:
you mean: the fields that have just been updated?
technically, that's not needed... they should be the same as in the current object.
in a defensive scenario: i'd be more concerned about "old_data" being modified on its way through the plugins.